pkg/streamingpromql/engine_test.go

```diff
@@ -1885,6 +1892,10 @@ func TestCompareVariousMixedMetricsRate(t *testing.T) {
 		expressions = append(expressions, fmt.Sprintf(`rate(series{label=~"(%s)"}[1m])`, labelRegex))
 		expressions = append(expressions, fmt.Sprintf(`sum(rate(series{label=~"(%s)"}[2m15s]))`, labelRegex))
 		expressions = append(expressions, fmt.Sprintf(`sum(rate(series{label=~"(%s)"}[5m]))`, labelRegex))
+		expressions = append(expressions, fmt.Sprintf(`increase(series{label=~"(%s)"}[45s])`, labelRegex))
+		expressions = append(expressions, fmt.Sprintf(`increase(series{label=~"(%s)"}[1m])`, labelRegex))
+		expressions = append(expressions, fmt.Sprintf(`sum(increase(series{label=~"(%s)"}[2m15s]))`, labelRegex))
+		expressions = append(expressions, fmt.Sprintf(`sum(increase(series{label=~"(%s)"}[5m]))`, labelRegex))
```

Comment:
How about adding a second loop over `rate` and `increase` rather than duplicating these lines?